mnefun/_mnefun.py

```diff
@@ -2330,3 +2332,114 @@ def plot_reconstruction(evoked, origin=(0., 0., 0.04)):
     axs[2, 0].set_ylabel('Reconstruction')
     fig.tight_layout()
     return fig
+
+
+def plot_chpi_snr_raw(raw, winlen, nharm, stop=None):
+    """Compute and plot cHPI SNR from raw data
+
+    Parameters
+    ----------
+    winlen : int
```

Comment:
It is already in seconds (which should be mentioned in the docstring) and gets converted to samples.
I don't like `duration` that much, isn't `winlen` more descriptive? (Maybe it should be `window_len`)